README.md

```diff
@@ -1,3 +1,50 @@
+# CFDE ICC Evaluation Core
+
+## Overview
+
+The [Common Fund Data Ecosystem (CFDE)](https://commonfund.nih.gov/dataecosystem) is an effort to bring together knowledge across [Common Fund](https://commonfund.nih.gov/) programs into a cohesive resource.
+The Integration and Coordination Center (ICC) is a center within the CFDE responsible for, among other things, evaluating the activities of Common Fund programs.
+
+This repository is a place to:
+
+- View a high-level evaluation of Common Fund program activity (e.g. projects over time, dollars awarded, # of publications produced, etc.).
+- Submit details about your Common Fund project to be included in the above.
+- Maintain code used to coordinate the above.
+
+The evaluation can be viewed as a live webapp dashboard or as separate PDF reports.
+
+[🖱️ The Webapp](cfde-eval.netlify.app)
+
+[📜 PDF Reports](cfde-eval.netlify.app/reports)
+
+## Submit your project
+
+We are able to get most details about Common Fund projects automatically from NIH resources, but there are some pieces of info that you must submit manually so that we have everything we need to create the evaluation.
```

Comment:
```suggestion
We gather details about Common Fund projects automatically from NIH resources, but there are some pieces of info that require manual intervention to be integrated.
```

Reply:
I've commited something similar to this.